AppServer/google/appengine/api/logservice/logservice_stub.py

```diff
@@ -115,21 +117,28 @@ def _get_log_server(self, app_id, blocking):
       return key, queue.get(False)
     except Empty:
       pass
-    if os.path.exists(LogServiceStub._LOGSERVER_PATH):
-      client = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
-      try:
-        client.connect(LogServiceStub._LOGSERVER_PATH)
-        client.setblocking(blocking)
-        client.send('a%s%s' % (struct.pack('I', len(app_id)), app_id))
-        return key, client
-      except socket.error:
-        return None, None
-    return None, None
+    client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    #get head node_private ip from /etc/appscale/head_node_private_ip
+    ip = file_io.read("/etc/appscale/head_node_private_ip").rstrip()
+    try:
+      client.connect((ip, 7422))
+      client.setblocking(blocking)
+      client.send('a%s%s' % (struct.pack('I', len(app_id)), app_id))
+      return key, client
+    except socket.error, e:
+      logging.info("Log Server at {ip} refused connection".format(ip=ip))
```

Comment:
We are not using the e variable.
Maybe also better to use logging.exception, then we are sure that the real problem is logged, because now we are hiding the real exception.